Atomic/discovery.py

```
@@ -52,7 +53,10 @@ def assemble_fqdn(self, include_tag=True, registry=None):
         fqdn = fqdn if not self.repo else "{}/{}".format(fqdn, self.repo)
         fqdn += "/{}".format(self.image)
         if include_tag:
-            fqdn += ":{}".format(self.tag)
+            if self.tag:
```

Should we raise an error if both tag and digest are defined at the same time?